tools/rpkg/src/utils.cpp

```diff
@@ -198,6 +198,21 @@ SEXP RApiTypes::ValueToSexp(Value &val) {
 		Rf_setAttrib(res, Rf_install("tzone"), r_ts.Protect(Rf_mkString("UTC")));
 		return res;
 	}
+	case LogicalTypeId::TIME: {
+		res = r.Protect(NEW_NUMERIC(1));
+		double *dest_ptr = ((double *)NUMERIC_POINTER(res));
+		dest_ptr[0] = ((double)val.value_.time.micros) / 1000;
+		NUMERIC_POINTER(res)[0] = val.value_.time.micros;
```

Comment:
This looks odd to me. Line 205 looks like it repeats what lines 203 and 204 are doing, but with a different value. 

Dividing by 1000 looks right to me, the current code seems to do it wrong. How is this code tested?